tooling/nargo_fmt/tests/input/trait.nr

```diff
@@ -0,0 +1,2 @@
+trait Foo: Bar + Baz {}
```

Comment:
Assuming this is a TODO but I'll comment anyway that the input file is the same as the expected file

Reply:
Oh, it seems traits are formatted "as-is" (without changes), so not really formatted. I wrote the test before realizing this, but I thought it was still good to have it.